.github/workflows/github-actions-clang-tidy.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           config_file: '.clang-tidy'
           exclude: "*/codeGenerator/templates/*"
           split_workflow: true
+          apt_packages: libomp-15-dev
```

Comment:
The Dockerfile in the action installs clang 14 and 15. Let's err in the side of caution and install both for omp as well?

Reply:
you got it!, just added it.

Reply:
we're now getting a
`
The following packages have unmet dependencies:
   libomp-14-dev : Conflicts: libomp-x.y-dev
`

and that's because development packages (-dev) install header files (like omp.h) into standard system locations (e.g., /usr/include). If you tried to install both versions, they would attempt to place their potentially different omp.h files (and other headers) in the same locations, leading to an inconsistent and broken state. apt prevents this conflict by refusing to install both.

Because of that I will have to remove `libomp-14-dev`

Reply:
This is only for the purpose of clang-tidy finding the header as we are not compiling any code here.  I don't think 14 vs 15 will matter.